build.sbt

```diff
@@ -1,76 +1,144 @@
+      "com.typesafe.akka"                       %% "akka-actor"                     % defaultVersion % "provided",
+      "io.circe"                                %% "circe-core"                     % defaultVersion,
+      "org.apache.logging.log4j"                %% "log4j-api-scala"                % "11.0",
       // NOTICE: This is added because of the exclusion rules on "elasticsearch-cluster-runner".
       //         While it is important to exclude those libs because of clients of this apso lib, our tests
       //         require the presence of the netty dependencies.
-      "io.netty"                                   % "netty-all"                      % "4.1.43.Final"   % "test",
-      "com.typesafe.akka"                         %% "akka-http-testkit"              % "10.1.11"        % "test",
-      "com.typesafe.akka"                         %% "akka-stream-testkit"            % "2.6.0"          % "test",
-      "junit"                                      % "junit"                          % "4.12"           % "test",
-      "net.ruippeixotog"                          %% "akka-testkit-specs2"            % "0.2.3"          % "test",
-      "org.apache.logging.log4j"                   % "log4j-core"                     % "2.12.1"         % "test",
-      "org.scalacheck"                            %% "scalacheck"                     % "1.14.2"         % "test",
-      "org.specs2"                                %% "specs2-core"                    % "4.8.1"          % "test",
-      "org.specs2"                                %% "specs2-scalacheck"              % "4.8.1"          % "test",
-      "org.specs2"                                %% "specs2-junit"                   % "4.8.1"          % "test"))
+      "io.netty"                                 % "netty-all"                      % defaultVersion % "test",
+      "com.typesafe.akka"                       %% "akka-http-testkit"              % defaultVersion % "test",
+      "com.typesafe.akka"                       %% "akka-stream-testkit"            % defaultVersion % "test",
+      "junit"                                    % "junit"                          % "4.12"         % "test",
+      "net.ruippeixotog"                        %% "akka-testkit-specs2"            % defaultVersion % "test",
+      "org.apache.logging.log4j"                 % "log4j-core"                     % "2.11.2"       % "test",
+      "org.scalacheck"                          %% "scalacheck"                     % defaultVersion % "test",
+      "org.specs2"                              %% "specs2-core"                    % defaultVersion % "test",
+      "org.specs2"                              %% "specs2-scalacheck"              % defaultVersion % "test",
+      "org.specs2"                              %% "specs2-junit"                   % defaultVersion % "test"))
 
 lazy val testkit = project.in(file("testkit"))
```

Comment:
Can't the `module` helper be used here?

Reply:
Yes, changed on 815fc9f6e11c0251540c4c31a34cbed6f94ebd2b.